LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 Google Inc. All Rights Reserved.
+   Copyright 2016 Google Inc. All Rights Reserved. Content by The Keptn Authors
```

Comment:
This is weird... Why do we attribute Google a copyright of this repository? Is it best practice that we have Google Code Labs in our repo, and the docs are kind of a 2nd class citizen? Should we restructure the repository?

Reply:
Based on a quick search I found: https://softwareengineering.stackexchange.com/questions/304874/declaring-multiple-licences-in-a-github-project

Reply:
Thanks, I'll take a look and do some changes accordingly